Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Use the Python 3.11 slim image as the base image
+FROM python:3.11-slim
+
+# Set the working directory inside the container
+WORKDIR /app
+
+# Enable non-interactive mode for debconf
+RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
+
+# Update the package repository and install required dependencies
+RUN apt-get update -y && \
+    apt-get install -y build-essential cmake unzip pkg-config \
+    libjpeg-dev libpng-dev libtiff-dev \
+    libavcodec-dev libavformat-dev libswscale-dev libv4l-dev \
+    libatlas-base-dev gfortran && \
```

Comment:
Can you check [this comment](https://github.com/Udayraj123/OMRChecker/pull/152/files#r1359375303)? I think you need to upload a video of [CropOnMarkers sample](https://github.com/Udayraj123/OMRChecker/blob/master/samples/sample1/template.json) working (sample1)

Reply:
Working on myside for CropOnMarkers , also added in video below